Iryna_Ramanets/Module_1_Java/Java_Fundamentals/HW3/src/com/epam/cdp/chef/interfaces/ICaloriesCalculatable.java

```diff
@@ -0,0 +1,10 @@
+package com.epam.cdp.chef.interfaces;
```

Comment:
I recommend following structure of packages
com.epam.cdp.chef.model with classes ICaloriesCalculatable
com.epam.cdp.chef.model.salad
com.epam.cdp.chef.model.vegetable